### ISSUE TITLE
Error on dependency parsing by JSPM

### ISSUE THREAD

**Issue by Weedshaker** [webtorrent/webtorrent#1481]
**0.102.1**

**Ubuntu 16.04**

**Chrome, Chromium, Firefox -> latest**

**What did you expect to happen?**
sfx bundle

**What actually happened?**
Error on dependency parsing for github:webtorrent/webtorrent@0.102.1/webtorrent.min.js at file:///.../jspm_packages/github/webtorrent/webtorrent@0.102.1/webtorrent.min.js
=> MultipleErrors: <compiler-parse-input>:1:98229: 'this' is not allowed before super()

**Comment by KayleePop** [comment]
Here's what threw the error from the minified build using that line:column from the trace (beautified)

```js
class o extends r.Writable {
  constructor(e, t, r) {
    function o(t) {
      this.destroyed || (e.put(i, t), i += 1)
    }

    if (super(r), r || (r = {}), !e || !e.put || !e.get) {
      throw new Error("First argument must be an abstract-chunk-store compliant store");
    }

    if (t = +t, !t) throw new Error("Second argument must be a chunk length");

    this._blockstream = new n(t, {
      zeroPadding: !1
    }), this._blockstream.on("data", o).on("error", e => {
      this.destroy(e)
    });

    let i = 0;

    this.on("finish", function() {
      this._blockstream.end()
    })
  }
```

This is clearly from here https://github.com/feross/chunk-store-stream/blob/v3.0.0/write.js

I think the function onData() ([here](https://github.com/feross/chunk-store-stream/blob/v3.0.0/write.js#L23)) is getting pushed up above `super(opts)` by the minifier.

**Comment by feross** [comment]
Published the fix in chunk-store-stream 3.0.1

**Comment by DiegoRBaquero** [comment]
@feross missed the other PR with a minor change :p

**Comment by KayleePop** [comment]
Is this a bug in the minifier? Should we open an issue?

**Comment by DiegoRBaquero** [comment]
@KayleePop Nope :)

Fix published as 0.102.2 :)

**Comment by Weedshaker** [comment]
Thanks for the quick fix!!!
